src/main/webapp/index.html

```diff
@@ -22,7 +22,16 @@ <h1>
           Discover the world's hidden gems <br />
           before planning your next trip.
         </h1>
-        <input type="text" name="search" placeholder="Search a city, country, etc." />
+
+        <!-- Search Bar -->
+        <form autocomplete="off">
+          <div class="autocomplete" action="/SearchResults.html" method="GET">
+            <input id="myInput" type="text" name="q" placeholder="Search a country" />
+            <div id="autocomplete-list" class="autocomplete-items"></div>
+          </div>
+          <!-- <input type="submit" value="Go" /> -->
```

Comment:
Maybe uncomment but hide so the check passes? (I don't know if it will)
```suggestion
          <input type="submit" value="Go" hidden />
```